app/uk/gov/hmrc/timetopayproxy/connectors/TtpTestConnector.scala

```diff
@@ -45,7 +45,7 @@ trait TtpTestConnector {
 
 @Singleton
 class DefaultTtpTestConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient) extends TtpTestConnector {
-
+  @annotation.nowarn
```

Comment:
Is it possible to fix the warnings rather than ignore them? (Not sure what the warning was so wanted to ask)

Reply:
Hi, for this one, I've spoken with @Jxr227 and he confirmed this specific class  is used in ET to facilitate e2e testing and are not considered production code.

Reply:
Could we put it in a testOnly package like ifs ?

Reply:
Sounds like a good idea @johnregan. Otherwise it confuses the reader what and where that is used. I'm going to create a separate ticket for it. Thanks

Reply:
Created the ticket https://jira.tools.tax.service.gov.uk/browse/DTD-1153 regarding @johnregan 's comment.